### ISSUE TITLE
[Enhancement] Falling-effect life option for high-power systems

### ISSUE THREAD

**Issue by P-Star7** [Wohlstand/TheXTech#321]
All SMBX falling-effects (the death of enemies like Koopas, Bullet Bills, Larry, etc.) have a limited "life" presumably to save on RAM. This is visible in vertical levels where the player can easily vertically follow the effect. Now that technology has advanced and SMBX is more efficient than ever in RAM usage anyways, I propose an .ini setting to determine how long their life is, or at least a boolean that makes them last much longer.

This would be forcefully disabled at compile-time on low-power systems using the low-power compile flag.

**Comment by ds-sloth** [comment]
I agree that this should be disabled at compile-time on performance-sensitive systems, but I don't think we need to use an ini setting. Instead, we can directly detect when the effect is no longer onscreen. Easier for developers, and almost always yields the expected effect.
